add full character substitution details (#6186)
umbraco/UmbracoDocs: umbraco-heartcore/release-notes/core-dependency-upgrade.md

**This is a breaking change.** Unless your application has been built with redirection in mind, then external links to affected content items may no longer work post-migration.
{% endhint %}

> The below table shows a comparison of new and old behaviour for all changes. This is not a list of all characters that are substituted, rather it is a list of those that are changing behaviour only.

Reviewer: A suggestion for a slight rewrite, as well as using US English instead of UK.
Let me know what you think:

```suggestion
The following table compares new and old behavior for all changes. This is not a list of all characters that are substituted, rather it is a list of those with changed behavior.
```

Author: Looks good to me!
